### PR TITLE
Sfno fix

### PR DESCRIPTION
**Pull request by daviddpruitt**
# Modulus Pull Request

## Description
Directly copy pytorch patchs instead of monkeypatching

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is up to date with these changes.
- [ ] An [issue](https://github.com/NVIDIA/modulus/issues) is linked to this pull request.

### CODE REVIEW

**Review by mnabian — APPROVED** [review]
LGTM

**Review by Alexey-Kamenev — APPROVED** [review]
LGTM.

File: modulus/experimental/sfno/patch_pytorch.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+cp third_party/torch/distributed/utils.py /usr/local/lib/python3.10/dist-packages/torch/distributed/
```
Comment (akshaysubr) [inline modulus/experimental/sfno/patch_pytorch.sh:2]: Why a hardcoded python path here?

**Pull request merged**